sdk/python/kfp/dsl/_container_op.py

```diff
@@ -198,14 +200,14 @@ def set_cpu_limit(self, cpu):
     return self.add_resource_limit("cpu", cpu)
 
   def set_gpu_limit(self, gpu, vendor = "nvidia"):
-    """Set gpu limit for the operator. This function add '<vendor>.com/gpu' into resource limit. 
-    Note that there is no need to add GPU request. GPUs are only supposed to be specified in 
+    """Set gpu limit for the operator. This function add '<vendor>.com/gpu' into resource limit.
```

Comment:
Looks like a stray change.

Reply:
Sorry I guess my text editor removed all the trailing whitespace. I reverted this change in my latest commit.